Makefile

```diff
@@ -80,10 +80,10 @@ release-install: release-container
 
 verify-shell: docker-running
 	docker run -ti --rm \
-		--privileged \
-		--volume /home/cockpit:/home/user \
+		--privileged --uts=host \
+		--volume /opt/cockpit-verify-secrets:/secrets \
```

Comment:
No, docker manpage says "You can add :ro or :rw suffix to a volume to mount it  read-only or read-write mode, respectively. By default, the volumes are mounted read-write.". If that weren't so, you couldn't chmod the files :-)